Commands/UpdateStoreCommand.php

```diff
@@ -116,6 +122,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $output->writeln(sprintf('Theme for shop "%s (#%d)" has been set to "%s".', $shop->getName(), $shop->getId(), $shop->getTemplate()->getTemplate()));
         }
 
+        if ($input->getOption('secure')) {
+            $shop->setSecure(true);
```

Comment:
Should we add a fallback for earlier versions or should we increase the minimum Shopware version to 5.3?

Reply:
I've added a fallback for setting `alwaysSecure`